raiden/api/python.py

```diff
@@ -847,3 +901,23 @@ def create_monitoring_request(
         # sign RequestMonitoring and return
         monitor_request.sign(self.raiden.signer)
         return monitor_request
+
+    def get_pending_transfers(self, token_address=None, partner_address=None):
+        chain_state = views.state_from_raiden(self.raiden)
+        transfer_tasks = views.get_all_transfer_tasks(chain_state)
+
+        if token_address is not None and partner_address is not None:
+            try:
+                partner_channel = self.get_channel(
+                    registry_address=self.raiden.default_registry.address,
+                    token_address=token_address,
+                    partner_address=partner_address,
+                )
+            except ChannelNotFound:
+                # just return an empty list if queried for a nonexisting channel
+                return list()
```

Comment:
I think this should Raise the exception to the REST interface where the response should be 404.